package.json

```diff
@@ -37,6 +37,7 @@
     "@polkadot/util": "^6.0.5",
     "@polkadot/util-crypto": "^6.0.5",
     "@polkadot/wasm-crypto": "^4.0.2",
+    "warthog": "https://github.com/ondratra/warthog/releases/download/v2.23.1/warthog-v2.23.1.tgz",
```

Comment:
Is this published version that depends on your modified `graphql-playground` package?

Reply:
I managed to merge changes to Joystream's fork of Warthog repo. My own repository is no longer required, and https://github.com/Joystream/warthog/releases/download/v2.37.0/joystream-warthog-v2.37.0.tgz is used instead.